configs/ts7180_defconfig

```diff
@@ -18,7 +18,7 @@ CONFIG_OF_BOARD_SETUP=y
 CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/spl_sd.cfg"
 CONFIG_BOOTDELAY=1
 CONFIG_USE_PREBOOT=y
-CONFIG_PREBOOT="if test $jpuboot = 'on'; then setenv bootdelay -1; fi"
+CONFIG_PREBOOT="if test $jpuboot = 'on'; then setenv bootdelay -1; fi; if test $jpnochrg = 'on'; then env delete silo_chrg_pct; fi"
```

Comment:
Discussed using a value of `-1` to denote to not enable supercaps rather than deleting the variable.

Reply:
Fixed.